sql/src/main/java/io/crate/execution/engine/PhasesTaskFactory.java

```diff
@@ -61,6 +61,10 @@ public PhasesTaskFactory(ClusterService clusterService,
     }
 
     public ExecutionPhasesTask create(UUID jobId, List<NodeOperationTree> nodeOperationTreeList) {
+        return create(jobId, nodeOperationTreeList, false);
+    }
+
+    public ExecutionPhasesTask create(UUID jobId, List<NodeOperationTree> nodeOperationTreeList, boolean enableProfiling) {
```

Comment:
maybe create a named method instead of adding a boolean parameter. `taskFactory.withProfiling(job, nodeOps)` is more readable than `create(..., true)`

Reply:
I also like separate method names more, however we hardly do it anywhere. We should start doing that instead of method overloading.

Reply:
On the other hand, it's quite handy when passing boolean arguments...